avocado/utils/process.py

```diff
@@ -98,6 +98,18 @@ def __str__(self):
             return "CmdError"
 
 
+def sudo_available():
+    """
+    :return: True when sudo is available (or is root)
+    """
+    if os.getpid() == 0:
+        return True
+    elif system_output("id -u", ignore_status=True, sudo=True).strip() == "0":
```

Comment:
Why this line when `python -c 'import os; print os.getuid()'` gives the same result (and it's already being used)?